script.js

```diff
@@ -20,3 +20,69 @@ function myFunction(element) {
 
 const menuSections = document.querySelectorAll('.menu-sel');
 menuSections.forEach(myFunction);
+
+// **************Popup*******************
+const projectsInfo = {
+  name: 'Project name goes here',
+  img: 'imgs/Union close.png',
+  tech: ['HTML/CSS', 'Ruby on Rais', 'JavaScript'],
+  description:
+    'Lorem ipsum dolor sit amet consectetur adipiscing elitsed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam quis nostrud exercitation ullamco laboris nisiUt aliquip ex ea commodo consequat  Lorem ipsum dolor sit amet consectetur adipiscing elit sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam quis nostrud exercitation ullamco laboris nisi.',
+  liveLink: 'https://codehass.github.io/Portfolio/',
+  sourceLink: 'https://github.com/codehass/Portfolio/tree/mobil-menu',
+};
+
+function popupProjects(project) {
+  project.addEventListener('click', () => {
+    const newDiv = document.createElement('div');
+    newDiv.innerHTML += `
+        <div class ="desk-popup">
+          <div class ="top-popup" >
+          <img src="${projectsInfo.img}" class ="menuClose" id ="menuClose" />
+          <p class ="popup-title"> ${projectsInfo.name} </p>
+          </div>
+          <ul>
+              <li>${projectsInfo.tech[0]}</li>
+              <li>${projectsInfo.tech[1]}</li>
+              <li>${projectsInfo.tech[2]}</li>
+          </ul>
+          <img class ="desimg"  src="imgs/Group 110.png"  />
+          <div>
+            <p class ="popup-descp">${projectsInfo.description}</p>
+          </div>
+          <div class ="buttPopup">
+              <div class ="buttPopupflex">
+                <a class ="popup-butt1" href="${projectsInfo.liveLink}" >See live</a>
+                <img  class ="popup-butt1img" src="imgs/Union (1).svg" />
+              </div>
```

Comment:
- The desktop version of your portfolio has some little differences from the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this 👍 👍 

Your project:
<img width="960" alt="wrgwrg" src="https://user-images.githubusercontent.com/73607512/202404929-8e416313-7427-4e68-8edd-3955f86d2c9e.PNG">
<img width="947" alt="wwsrwr" src="https://user-images.githubusercontent.com/73607512/202404976-0b8a3add-846c-4ea4-a3d6-8b0ce175d26d.PNG">

Figma template:
<img width="486" alt="gfefef" src="https://user-images.githubusercontent.com/73607512/202405042-96292b08-3a5c-4f3f-8536-d161bd045771.PNG">
<img width="475" alt="rwrgwrg" src="https://user-images.githubusercontent.com/73607512/202405074-280067c7-7f56-47e9-96fd-2bd02f7e4d93.PNG">